test/powershell/Modules/Microsoft.PowerShell.Management/Get-Item.Tests.ps1

```diff
@@ -1,4 +1,12 @@
 Describe "Get-Item" -Tags "CI" {
+    BeforeAll {
+        if ( $IsWindows ) {
+            $skip = $false
```

Comment:
Instead of calling it `$skip`, I think it would be better to call it `$skipNotWindows`